localization/replace/english/wc_dlc_fp1_l_english.yml

```diff
@@ -52,6 +52,9 @@
  NORDIC_STATE_TITLE_NAME:0 "Nornir [target_duchy.GetNameNoTierNoTooltip]"
  NORSE_STATE_TITLE_NAME:0 "Vrykul [target_duchy.GetNameNoTierNoTooltip]"
 
- scandi_adventurer_event_troops:0 "Vrykul Adventurers"
+ scandi_adventurer_event_troops:1 "Adventurers"
 
- SteleCommemoratesAncestors_Gluttonous:0 "$SteleCommemoratesAncestors_Intro$. Not once in [second.MakeScope.Var('stele_container').Char.Custom('GetCustomAge')] did [second.MakeScope.Var('stele_container').Char.GetSheHe] skip a meal. $SteleCommemoratesCarver$."
\ No newline at end of file
+ SteleCommemoratesAncestors_Gluttonous:0 "$SteleCommemoratesAncestors_Intro$. Not once in [second.MakeScope.Var('stele_container').Char.Custom('GetCustomAge')] did [second.MakeScope.Var('stele_container').Char.GetSheHe] skip a meal. $SteleCommemoratesCarver$."
+ 
+ fp1_reprieve_from_sa_modifier:0 "Reprieve from Adventurers"
+ fp1_reprieve_from_sa_modifier_desc:0 "Somehow, this character has won a measure of respite from the relentless raiding hordes.\n$EFFECT_LIST_BULLET$ Will not be attacked by landless Adventurers"
```

Comment:
Change the wording, it just doesn't sound very good when I read it out and the heading of the event is "Reprieve from Adventurers" so it makes sense for the character to win a reprieve rather than respite:

"Somehow, this character has won a reprieve from the relentless invading hordes Having proven their strength, .\n$EFFECT_LIST_BULLET$ will not be attacked by ambitious adventurers for some time."

Reply:
```suggestion
 fp1_reprieve_from_sa_modifier_desc:0 "Somehow, this character has won a reprieve from the relentless invading hordes Having proven their strength, .\n$EFFECT_LIST_BULLET$ will not be attacked by ambitious adventurers for some time."
```